test/index.js

```diff
@@ -243,7 +248,12 @@ describe('hapi-setup Plugin', function () {
           },
           attributes: {
             name: 'bar',
-            version: '1.0.0'
+            version: '1.0.0',
+            pkg: {
+              version: '0.0.0'
+            },
+            multiple: false,
```

Comment:
I had to add them to make the test pass.  So perhaps the issue is how the test works.  Again I'm making assumptions on how this should have worked considered all tests where failing.